website/content/en/docs/olm-integration/legacy/quickstart-bundle.md

```diff
@@ -0,0 +1,145 @@
+...
+```
+
+**Note:** By default, `olm status` and `olm uninstall` auto-detect the OLM version installed in your cluster.
+This can fail if the installation is broken in some way, so the version of OLM can be overridden using the
+`--version` flag provided with these commands.
+
+## Creating a bundle
+
+_If working with package manifests, see the [package manifests quickstart][quickstart-package-manifests]._
+
+We will now create bundle manifests and metadata by running `generate bundle` in the root of the memcached-operator project.
+
+```sh
+$ operator-sdk generate bundle --version 0.0.1
+```
+
+A bundle manifests directory `deploy/olm-catalog/memcached-operator/manifests` containing a CSV and all CRDs
+in `deploy/crds`, a bundle [metadata][bundle-metadata] directory `deploy/olm-catalog/memcached-operator/metadata`,
+and a [Dockerfile][bundle-dockerfile] `bundle.Dockerfile` have been created in the Operator project.
+
+These files can be statically validated by `bundle validate` to ensure the on-disk bundle representation is correct:
+
+```console
+$ operator-sdk bundle validate ./deploy/olm-catalog/memcached-operator
+INFO[0000] Found annotations file                        bundle-dir=deploy/olm-catalog/memcached-operator container-tool=docker
+INFO[0000] Could not find optional dependencies file     bundle-dir=deploy/olm-catalog/memcached-operator container-tool=docker
+INFO[0000] All validation tests have completed successfully
+```
+
+## Deploying an Operator with OLM
+
+At this point in development we've generated all files necessary to build the memcached-operator bundle.
+Now we're ready to test and deploy the Operator with OLM.
+
+### Deploying bundles in production
+
+OLM and Operator Registry consumes Operator bundles via an [index image][index-image],
+which are composed of one or more bundles. To build a memcached-operator bundle, run:
+
+```console
+$ docker build -f bundle.Dockerfile -t quay.io/<username>/memcached-operator:v0.1.0 .
+```
+
+Although we've validated on-disk manifests and metadata, we also must make sure the bundle itself is valid:
+
+```console
+$ docker push quay.io/<username>/memcached-operator:v0.1.0
+$ operator-sdk bundle validate quay.io/<username>/memcached-operator:v0.1.0
+INFO[0000] Unpacked image layers                         bundle-dir=/tmp/bundle-716785960 container-tool=docker
+INFO[0000] running docker pull                           bundle-dir=/tmp/bundle-716785960 container-tool=docker
+INFO[0002] running docker save                           bundle-dir=/tmp/bundle-716785960 container-tool=docker
+INFO[0002] All validation tests have completed successfully  bundle-dir=/tmp/bundle-716785960 container-tool=docker
+```
+
+The SDK does not build index images; instead, use the Operator package manager tool [`opm`][opm] to manage
+index images. Once one has been built, follow the index image [usage docs][doc-olm-index] to add an index to a
```

Comment:
```suggestion
The SDK does not build index images; instead, use the Operator package manager tool [`opm`][opm] to [build an index image][index-image]. Once one has been built, follow the index image [usage docs][doc-olm-index] to add an index to a
```

Slight nit:
I think we should link to the doc on how to build the index image here now that we have a bundle.
https://github.com/operator-framework/operator-registry#building-an-index-of-operators-using-opm

We've got a link above but that comes before we've even built the bundle and diverts the flow of the doc.

Reply:
Good point, updated links and wording.